tardis/montecarlo/base.py

```diff
@@ -5,14 +5,19 @@
 from astropy import units as u
 from tardis import constants as const
 from numba import set_num_threads
+from numba.cuda import is_available
 
 from scipy.special import zeta
 from tardis.montecarlo.spectrum import TARDISSpectrum
 
 from tardis.util.base import quantity_linspace
 from tardis.io.util import HDFWriterMixin
 from tardis.montecarlo import packet_source as source
-from tardis.montecarlo.montecarlo_numba.formal_integral import FormalIntegrator
+if is_available():
+    from tardis.montecarlo.montecarlo_numba.formal_integral_cuda import FormalIntegrator
+elif not is_available():
```

Comment:
You can just do `else` here instead of calling the `is_available()` function again

Reply:
That would import the entire cuda library into this file, should the one function that is used just be called instead? I did the elif to make sure I would catch the false case. I didn't think there was more than two, but I can change it to just be else.